spec_7/dateConverter.js

```diff
@@ -0,0 +1,63 @@
+import { identifiers, months } from './resources'
+import fs from 'fs'
+
+const converter = {
+
+  getRawDates: filePath => {
+    const raw_dates = fs.readFileSync( filePath, 'utf8' )
+    return raw_dates.split('\n')
+  },
+
+  trimCommas: word => word.replace( /,/g, '' ),
+
+  getWholeYear: twoDigits =>
+    parseInt( twoDigits ) > 49 ? '19'+twoDigits : '20'+twoDigits,
+
+  filterFormats: (date_array, identifier) =>
+    date_array.filter( date => date.indexOf(identifier) > -1 ),
+
+  makeDateString: valuesObj =>
+    `${valuesObj.year}-${valuesObj.month}-${valuesObj.day}`,
+
+  getValidFormats: dates => {
```

Comment:
For consistency with your other code, you should omit the braces and `return` statement.